docs/observability/aws/deploy-use-aws-observability/deploy-with-terraform.md

```diff
@@ -66,9 +66,15 @@ Before you run the Terraform script, perform the following actions on a server m
     ```bash
     $ terraform --version
     ```
-1. Install the latest version of [curl](https://curl.haxx.se/download.html).
+1. Install the latest version of [curl](https://curl.haxx.se/download.html). To check the installed curl version, run the following command:
+    ```bash
+    $ curl --version
+    ```
 1. Install [Python](https://www.python.org/) version 3.7 or later.
-1. Install the latest version of [jq](https://github.com/stedolan/jq/wiki/Installation) command-line JSON parser. This is required for running the `fields.sh` batch file.
+1. Install the latest version of [jq](https://github.com/stedolan/jq/wiki/Installation) command-line JSON parser. This is required for running the `fields.sh` batch file. To check the installed jq version, run the following command:
+    ```bash
+    $ jq --version
```

Comment:
```suggestion
    jq --version
```

Reply:
Hi @npande , while this is correct, we remove them in our Docs site so that customers can copy/paste and run our snippets without having to remove the `$` sign